articles/communication-services/quickstarts/includes/user-access-token-js.md

```diff
@@ -115,12 +115,11 @@ Access tokens are short-lived credentials that need to be reissued. Not doing so
 
 ## Refresh access tokens
 
-To refresh an access token, use the `CommunicationUser` object to reissue:
+Refreshing access tokens is as easy as calling `issueToken` with the same identity that was used to issue the tokens. You also need to provide the `scopes` of the refreshed tokens. 
 
-```javascript  
-// Value existingIdentity represents identity of Azure Communication Services stored during identity creation
-identityResponse = new CommunicationUser(existingIdentity);
```

Comment:
Readers should be aware, that in order to refresh access token, they must use CommunicationUser class. New instance of CommunicationUser can be created based on string value. We want to avoid the situation, that people create new identities just because they don't know how to refresh the token otherwise.

Reply:
`CommunicationUser` is not a class in js; just an interface. The response of `createUser` is a `CommunicationUser` so passing it to `issueToken` will refresh the tokens.